srcpkgs/rubberband/template

```diff
@@ -1,22 +1,40 @@
 # Template file for 'rubberband'
 pkgname=rubberband
-version=3.1.2
+version=3.2.1
 revision=1
 build_style=meson
+configure_args="--buildtype=release -Dfft=fftw"
```

Comment:
Any reason to set the buildtype here since that overwrites what the meson build style does, which is does afaik to keep the xbps-src build flags?

Reply:
Without passing this, `meson.build` was detecting the build type as "Debug" instead of "Release."

Reply:
I added a patch to fix the detection, and I'm also going to submit it upstream.